src/main/java/com/minecolonies/client/gui/WindowHutFarmer.java

```diff
@@ -264,7 +81,6 @@ private void nextClicked(Button ignored)
     public void onButtonClicked(Button button)
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this method to simply inherit it. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS1185)